kotlin-audio/src/main/java/com/doublesymmetry/kotlinaudio/notification/NotificationManager.kt

```diff
@@ -138,46 +159,31 @@ class NotificationManager internal constructor(private val context: Context, pri
     var showPreviousButtonCompact: Boolean
         get() = internalManager?.usePreviousActionInCompactView ?: false
         set(value) {
-            internalManager?.usePreviousActionInCompactView = value
+            scope.launch {
+                internalManager?.usePreviousActionInCompactView = value
+            }
         }
 
     init {
-        channelId = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
-            createNotificationChannel()
-        } else {
-            ""
-        }
-
-        if (!isJUnitTest()) {
-            mediaSessionConnector.setPlayer(exoPlayer)
+        scope.launch {
+            if (!isJUnitTest()) {
+                mediaSessionConnector.setPlayer(exoPlayer)
+            }
         }
     }
 
-    @RequiresApi(Build.VERSION_CODES.O)
-    private fun createNotificationChannel(): String {
-        val channelId = CHANNEL_ID
-        val channelName = context.getString(R.string.playback_channel_name)
-        val channel = NotificationChannel(channelId, channelName, NotificationManager.IMPORTANCE_LOW)
-        channel.description = "Used when playing music"
-        channel.setSound(null, null)
-
-        val service = context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
-        service.createNotificationChannel(channel)
-        return channelId
-    }
-
```

Comment:
We don't need to create out own notification channel, as ExoPlayer does that on its own.